include/LightGBM/c_api.h

```diff
@@ -306,21 +359,21 @@ DllExport int LGBM_BoosterGetPredict(BoosterHandle handle,
 /*!
 * \brief make prediction for file
 * \param handle handle
+* \param data_filename filename of data file
+* \param data_has_header data file has header or not
 * \param predict_type
 *          0:normal, with transform (if needed)
 *          1:raw score
 *          2:leaf index
-* \param n_used_trees number of used tree, < 0 means no limit
-* \param data_has_header data file has header or not
-* \param data_filename filename of data file
+* \param num_iteration number of iteration for prediction, < 0 means no limit
 * \param result_filename filename of result file
 * \return 0 when success, -1 when failure happens
 */
 DllExport int LGBM_BoosterPredictForFile(BoosterHandle handle,
-  int predict_type,
-  int64_t n_used_trees,
-  int data_has_header,
   const char* data_filename,
+  int data_has_header,
```

Comment:
using ```int``` to replace ```bool``` in c_api is more general, for some language may have different definition for ``bool``